### PR TITLE
[CM-1369] Add role-policy creation to secret center creation

### PR DESCRIPTION
**Pull request by Sieabah**

### CODE REVIEW

**Review by Robert-W — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -88,3 +88,19 @@ application via
...
+Secret Management
+--------------------------
+
+After your Deployment Center is created you can optionally create a secret center 
+which your containers can leverage for encrypted secrets. See 
+[Yadda-Secret](https://github.com/asymmetrik/yadda-secret) for the server package.
+
+To create the Secret Center you can run  
+
+	$ yadda create-secret-center <env>
```
Comment (andrewmarcus) [inline README.md:101]: Is the environment necessary for this?

File: lib/iam-tools.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+};
+
+/**
+ *
+ * @param {string} region - AWS Region
+ * @param {object|} iamOptions [undefined] - IAM Options
+ * @param {object} dynamoOptions [undefined] - DynamoDB Options
+ * @return {*}
+ */
+module.exports = function(region, iamOptions, dynamoOptions){
```
Comment (andrewmarcus) [inline lib/iam-tools.js:35]: set defaults maybe?  `iamOptions = null`

Comment (Sieabah) [reply lib/iam-tools.js:35]: Line 36 does just that.

File: README.md (hunk 1 above)
Comment (Sieabah) [reply README.md:101]: Yes as you can have different deployment centers per environment

**Pull request merged**